### PR TITLE
feat(elevation): add elevation 0dp

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: scripts/figma-parsers/fetch-figma-data.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ async function fetchData() {
 			response.on('data', chunk => receiver += chunk);
 			response.on('end', () => {
 				if (response.complete) {
-					fs.rmdirSync(RAW_DATA_PATH, { recursive: true });
+					fs.rmdirSync('./tmp', { recursive: true });
```
Comment (yinonov) [inline scripts/figma-parsers/fetch-figma-data.js:47]: mistake?

File: scripts/figma-parsers/parsers/elevation-figma-data-parser.js [hunk 2]
```diff
@@ -28,13 +29,17 @@ const getDocumentFragment = documentChild => documentChild.name.includes(CANVAS_
 
 function getDropShadowsFromFigmaConfig(dpSettings) {
     return `${dpSettings.effects.reduce((str, value) => {
-        str += `drop-shadow(${!value.offset.x ? value.offset.x : value.offset.x + 'px'} ${!value.offset.y ? value.offset.y : value.offset.y + 'px'} ${!value.radius ? value.radius : value.radius + 'px'} rgba(${value.color.r * 255}, ${value.color.g * 255}, ${value.color.b * 255}, ${value.color.a})) `
+        str += `drop-shadow(${!value.offset.x ? value.offset.x : value.offset.x + 'px'} ${!value.offset.y ? value.offset.y : value.offset.y + 'px'} ${!value.radius ? value.radius : value.radius + 'px'} rgba(${normalizeRGBValue(value.color.r)}, ${normalizeRGBValue(value.color.g)}, ${normalizeRGBValue(value.color.b)}, ${normalizeRGBValue(value.color.a, 1)})) `
```
Comment (yinonov) [inline scripts/figma-parsers/parsers/elevation-figma-data-parser.js:32]: maybe we want to adopt https://colord.omgovich.ru/?

File: scripts/figma-parsers/fetch-figma-data.js (hunk 1 above)
Comment (YonatanKra) [reply scripts/figma-parsers/fetch-figma-data.js:47]: The mistake was there before.

File: scripts/figma-parsers/parsers/elevation-figma-data-parser.js (hunk 2 above)
Comment (YonatanKra) [reply scripts/figma-parsers/parsers/elevation-figma-data-parser.js:32]: Might at some point. Will need to set it for all parsers I guess.

**Pull request merged**